docs/user-guide/data.md

```diff
@@ -50,13 +50,69 @@ Special syntax:
+The advanced search allows for the creation of complex queries using a graphical widget.
+The widget's contents are then turned into a string representing a Gravsearch (SPARQL) query to be sent to DSP-API.
 
-In addition, for a metadata field that is connected to another source type, it's possible to filter by this second source type.
+A query consists of the following elements:
+- ontology (data model) selection
+- selection of a resource class belonging to the selected ontology (optional)
+- specification of properties, comparison operators, and values (optional).
+
+Although selection of a resource or a property or both are optional, either a resource class has to be selected or at least one property has to be specified,
+otherwise the query is not considered valid and cannot be submitted.
 
 ![Search 2: Advanced search offers many filter combinations and is a powerful search tool.](../assets/images/search-advanced.png)*<https://admin.dasch.swiss/search/advanced> - Search 2: Advanced search offers many filter combinations and is a powerful search tool.*
 
-Currently, it is possible to search for only **one** source.
+#### Comparison Operators
+
+Depending on the value type of the chosen property,
+one or more of the following comparison operators can be selected:
+
+- `is equal to`: value equality: same number, exact same string, overlap of date periods, same target resource.
+- `is not equal to`: value inequality: not same number, not exact same string, no overlap of date periods, not same target resource.
+- `is greater than`: value comparison: number is greater than search value, date period begins after search value.
+- `is greater than or equal to` value equality / value comparison: number is equal to or greater than search value, overlap of date periods or date period begins after search value.
+- `is less than`: value comparison: number is less than search value, date period ends before search value.
+- `is less than or equal to`: value equality / value comparison: number is equal to or less than search value, overlap of date periods or date period ends before search value.
+- `exists`: value for the given property exists.
+- `is like`: search value is contained in a text using the SPARQL [REGEX](https://www.w3.org/TR/sparql11-query/#func-regex) function (supports regular expressions).
+- `matches`:
+  - text property: search value matches the text ([Lucene Query Parser Syntax](https://docs-api.dasch.swiss/08-lucene/lucene-query-parser-syntax/)).
+  - linking property: matches the specified linked resource.
+
+#### Search Examples
+
+##### Regular Expressions (is like)
+
+The `is like` operator lets the user search for texts that are *like* the search value via the support of regular expressions
```

Comment:
The term "*like*" will be formatted as italic; is that correct or should it display the asterisks as well?

Reply:
it's just formatting